test/load_brocfile_test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const loadBrocfile = require('../lib/load_brocfile');
+const chai = require('chai');
+
+const projectPath = 'test/fixtures/project';
+const brocfileFixture = require('../' + projectPath + '/Brocfile.js');
+
+describe('loadBrocfile', function() {
+  let oldCwd = null;
+
+  beforeEach(function() {
+    oldCwd = process.cwd();
+  });
+
+  afterEach(function() {
+    process.chdir(oldCwd);
+  });
+
+  context('with Brocfile.js', function() {
+    beforeEach(function() {
+      process.chdir(projectPath);
+    });
+
+    it('return tree definition', function() {
+      chai.expect(loadBrocfile()).to.equal(brocfileFixture);
+    });
+  });
+
+  context('with Brocfile.js, called in subfolder', function() {
+    beforeEach(function() {
+      process.chdir(projectPath + '/subdir');
+    });
+
+    afterEach(function() {
```

Comment:
i dont think this `afterEach` is required, as the parent describe blocks afterEach already restores the `chdir`

Reply:
it is needed as `load_brocfile` changes working dir https://github.com/broccolijs/broccoli/blob/master/lib/load_brocfile.js#L18

Reply:
but doesn't the outer `afterEach` already `chdir`? Am I confused and that one doesn't fire?

Reply:
right, sorry, it's pretty late here

Reply:
no worries